app/views/users/index.html.erb

```diff
@@ -1 +1,5 @@
-<h1>Users</h1>
\ No newline at end of file
+<%= "No users" if @user.empty? %>
+<% @user.each do |user| %>
+    <a href="<%= user_path(user) %>"><%= render user%></a>
+    <br>
+<% end %>
```

Comment:
[OPTIONAL] In scenarios like this it is a good practice to use rails partials. I helps keep your code DRY